include/oneapi/dpl/pstl/algorithm_fwd.h

```diff
@@ -836,7 +836,7 @@ __pattern_nth_element(__parallel_tag<_IsVector>, _ExecutionPolicy&&, _RandomAcce
 //------------------------------------------------------------------------
 // fill, fill_n
 //------------------------------------------------------------------------
-template <class _Tag, typename _ExecutionPolicy, typename _Tp, typename = void>
+template <class _Tag, typename _Tp, typename = void>
```

Comment:
It seems we can remove struct  `, typename = void`.

Reply:
Fixed

Reply:
This task doesn't correspondent with the goal of this PR.
So if you think that it's something extra - let's create separate issue.